### DISCUSSION TITLE
Ventola Fan SHIM rimane sempre accesa

### DISCUSSION THREAD

**Discussion by manny199** [indomus/forum#525] [GENERALE - Discussioni generiche]
Ciao a tutti,
ho seguito le istruzioni della guida riferita alla ventola Fan SHIM **da comandare tramite home assistant** ma, anche quando imposto la temperatura ad un livello superiore a quello della CPU, la ventola rimane comunque in funzione.
Anche azionando l'entità switch.fan_shim, posizionando in OFF, la ventola rimane comunque accesa.
Forse qualche problema relativo a GPIO?

**Comment by indomus** [comment]
E beh sì, o è installata male, o è rotta, o hai configurato male su Home Assistant. Usando lo switch DEVE rispondere.

**Comment by braudoo** [comment]
hai provato ad aggiungere "home assistant" alla lista di controllo della piedinatura fisica,
lanciando il comando

`sudo adduser homeassistant gpio`

Avevo avuto lo stesso problema e mi ero dimenticato di lanciare questo comando.

**Reply by manny199** [reply]
@braudoo Si, aggiunto a gpio ma rimane sempre accesa

**Reply by manny199** [reply]
Non ho però effettuato l'installazione del software fan SHIM su raspbian in quanto penso sia superfluo. Ho solo aggiunto homeassistant al gruppo gpio e inserito il file fan_shim_home_assistant.yaml in packages
